test/payments/multiReceiverTicketing.test.ts

```diff
@@ -509,16 +538,113 @@ describe('MultiReceiverTicketing', () => {
       syloTicketing.redeemMultiReceiver(
         ticket,
         redeemerRand,
-        { main: bobs[0].address, delegated: ethers.ZeroAddress },
+        bobs[0].address,
         senderSig,
         receiverSig,
       ),
     ).to.be.revertedWithCustomError(syloTicketing, 'TicketAlreadyRedeemed');
   });
 
+  it('can redeem for receiver using attached authorized account', async () => {
+    await stakingManager.addStake(toSOLOs(1), owner);
+    await setSeekerRegistry(
+      seekers,
+      registries,
+      seekerPowerOracle,
+      accounts[0],
+      accounts[1],
+      1,
+    );
+
+    await epochsManager.joinNextEpoch();
+    await epochsManager.initializeEpoch();
+
+    const alice = Wallet.createRandom();
+    const [bob] = await createFuturepassReceivers(1);
+    const delegatedWallet = Wallet.createRandom();
+
+    await syloTicketing.depositEscrow(toSOLOs(2000), alice.address);
+    await syloTicketing.depositPenalty(toSOLOs(50), alice.address);
+
+    const { ticket, redeemerRand, senderSig, ticketHash } =
+      await createWinningMultiReceiverTicket(
+        syloTicketing,
+        epochsManager,
+        alice,
+        owner,
+      );
+
+    const receiverSig = await createUserSignature(
+      ethers.getBytes(ticketHash),
+      SignatureType.Main,
+      bob,
+      delegatedWallet,
+      authorizedAccounts,
+    );
+
+    await syloTicketing.redeemMultiReceiver(
+      ticket,
+      redeemerRand,
+      bob.address,
+      senderSig,
+      receiverSig,
+    );
```

Comment:
```suggestion
    const receiverSig = await createUserSignature(
      ethers.getBytes(ticketHash),
      SignatureType.AttachedAuthorized,
      bob,
      delegatedWallet,
      authorizedAccounts,
    );

    console.log('receiver sig ', receiverSig);

    await syloTicketing.redeemMultiReceiver(
      ticket,
      redeemerRand,
      bob.address,
      senderSig,
      receiverSig,
    );
```
I think this is supposed to be attactched ?